src/manage_sql_nvts.c

```diff
@@ -1072,46 +1136,6 @@ get_tag (entity_t vt)
   else
     g_warning ("%s: no severities", __FUNCTION__);
 
-  child = entity_child (vt, "detection");
-  if (child)
-    {
-      if (strlen (entity_text (child)))
-        {
-          g_string_append_printf (tag,
-                                  "%svuldetect=%s",
-                                  first ? "" : "|",
```

Comment:
Because  these cases have been removed, the variable first is no longer used after the assignment on line 1031.  So the assignment is no longer needed, which is causing the build checks to fail.